loop/models.py

```diff
@@ -206,16 +222,26 @@ class Crop(LoopModel):
     measuring_unit = models.CharField(max_length=20, default="kg")
     is_visible = models.BooleanField(default=True)
     crop_name_en = models.CharField(max_length=30, null=True)
+
     def __unicode__(self):
         return self.crop_name
 
     class Meta:
         unique_together = ("crop_name",)
 
-
 post_save.connect(save_log, sender=Crop)
 pre_delete.connect(delete_log, sender=Crop)
 
+#############Crop name in multiple languages###############
+
+class Croplanguage(models.Model):
```

Comment:
please change it to CropLanguage and regenerate the migrations and make the required changes wherever table is used. Convention must be followed in models